docs/android/testing/screenshot_testing.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Android Screenshot Testing"
+sidebar_label: "Screenshot Testing"
+---
+
+🚧🚧🚧 Under Construction 🚧🚧🚧
```

Comment:
I kinda think we should remove the under construction parts. I think it will help us with followups

Reply:
I kept the one at the top because the rest is actually up to date with the current state of the impl of theses test by @dshokouhi. So I expect him or anyone to update this section when progress are going to be made.

Reply:
Hmm as we already have a test live maybe we can remove the under construction part and tweak as we need t?